Doc/tools/extensions/pyspecific.py

```diff
@@ -695,6 +675,8 @@ def setup(app):
     app.add_directive('audit-event', AuditEvent)
     app.add_directive('audit-event-table', AuditEventListDirective)
     app.add_directive('deprecated-removed', DeprecatedRemoved)
+    if sphinx.version_info[:2] < (7, 3):
+        app.add_directive('versionremoved', VersionChange)
```

Comment:
Hmm, I don't think this will work without `versionremoved` in `versionlabels`. Is it needed?

Reply:
Oh, true. Not strictly needed, but allows us to use the `.. versionremoved::` directive now, if we want to. I can remove from this PR if easier?

A